docs/user-manual/en/mfa.md

```diff
@@ -0,0 +1,44 @@
+# Multi Factor Authentication
+
+Kapua provides a Multi Factor Authentication feature (a.k.a. MFA), which allows a user to authenticate to a system only when he presents the 
```

Comment:
I think this should be change from "...a user to authenticate to a system..." to "...a user to authenticate within a system..." - what do you think?

Reply:
I agree that the "to" repetition is not very nice to see, however I'd prefer keeping this for now, since I think "authenticate to" is more common than "authenticate within".